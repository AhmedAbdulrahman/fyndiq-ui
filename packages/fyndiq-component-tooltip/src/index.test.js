import React from 'react'
import { shallow } from 'enzyme'

import Tooltip from './'

describe('fyndiq-component-tooltip', () => {
  test('should be rendered without props', () => {
    expect(shallow(<Tooltip text="help">tooltip</Tooltip>)).toMatchSnapshot()
  })

  test('should have the right position', () => {
    expect(shallow(
      <Tooltip text="help" position="tr">tooltip</Tooltip>
    )).toMatchSnapshot()
  })
})
