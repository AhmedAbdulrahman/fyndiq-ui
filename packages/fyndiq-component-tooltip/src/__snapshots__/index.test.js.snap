// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fyndiq-component-tooltip should be rendered without props 1`] = `
<Dropdown
  button={
    <span>
      tooltip
    </span>
  }
  hoverMode={true}
  margin={5}
  noArrow={false}
  noWrapperStyle={true}
  opened={false}
  position="bc"
  propagateClickEvent={false}
>
  <div
    className="
          tooltip
          position-bc
        "
  >
    help
  </div>
</Dropdown>
`;

exports[`fyndiq-component-tooltip should have a component for the prop text 1`] = `
<Dropdown
  button={
    <span>
      tooltip
    </span>
  }
  hoverMode={true}
  margin={5}
  noArrow={false}
  noWrapperStyle={true}
  opened={false}
  position="bc"
  propagateClickEvent={false}
>
  <div
    className="
          tooltip
          position-bc
        "
  >
    <div>
      hello
    </div>
  </div>
</Dropdown>
`;

exports[`fyndiq-component-tooltip should have the right position 1`] = `
<Dropdown
  button={
    <span>
      tooltip
    </span>
  }
  hoverMode={true}
  margin={5}
  noArrow={false}
  noWrapperStyle={true}
  opened={false}
  position="tr"
  propagateClickEvent={false}
>
  <div
    className="
          tooltip
          position-tr
        "
  >
    help
  </div>
</Dropdown>
`;
